platform/platform-app/src/main/resources/features.xml

```diff
@@ -776,6 +776,12 @@
         <bundle>mvn:ddf.platform.country/platform-country-converter-local/${project.version}</bundle>
     </feature>
 
+    <feature name="platform-usng4j" install="manual" version="${project.version}"
```

Comment:
@pklinef That may be the case, but I believe it's also the case the feature would get installed when the app is started.